python/samples/06.auth.oauth.messageExtensions/README.md

```diff
@@ -0,0 +1,61 @@
+# Teams Message Extension SSO Bot
```

Comment:
maybe we should remove the occurences of "SSO" in this README and replace with "oauth" to avoid confusion?